Validation/RecoMuon/python/track_selectors_cff.py

```diff
@@ -2,70 +2,79 @@
 
 # select probe tracks
 import PhysicsTools.RecoAlgos.recoTrackSelector_cfi
-probeTracks = PhysicsTools.RecoAlgos.recoTrackSelector_cfi.recoTrackSelector.clone()
-#probeTracks.quality = cms.vstring('highPurity')   # previous setting
-#probeTracks.quality = cms.vstring('loose')        # default
-probeTracks.tip = cms.double(3.5)
-probeTracks.lip = cms.double(30.)
-probeTracks.ptMin = cms.double(3.0)
-probeTracks.minRapidity = cms.double(-2.4)
-probeTracks.maxRapidity = cms.double(2.4)
+probeTracks = PhysicsTools.RecoAlgos.recoTrackSelector_cfi.recoTrackSelector.clone(
+#quality = 'highPurity'   # previous setting
+#quality = ['loose']        # default
+    tip = 3.5,
+    lip = 30.,
+    ptMin = 3.0,
+    minRapidity = cms.double(-2.4),
+    maxRapidity = cms.double(2.4)
```

Comment:
```suggestion
    minRapidity = -2.4,
    maxRapidity = 2.4
```

However, all these parameters are inside a `recoTrackSelectorPSet` : did you verify with an explicit dump whether they were actually modified? If so, then you should be also able to remove the explicit type from min/max rapidities

Reply:
@perrotta Yes, I have checked that! Explicit types can be removed from min/max rapidities. Somehow it got missed from my side.